hasura.planx.uk/tests/utils.js

```diff
@@ -1,7 +1,7 @@
 const fetch = require("isomorphic-fetch");
 
 async function gqlAdmin(query, variables = {}) {
-  const res = await fetch(`http://0.0.0.0:${process.env.HASURA_PORT}/v1/graphql`, {
+  const res = await fetch(`http://${process.env.HASURA_HOST}:${process.env.HASURA_PORT}/v1/graphql`, {
```

Comment:
I was trying to run this locally and have updated the environment variables here to enable that.